src/taipy/gui/utils/types.py

```
@@ -45,13 +45,13 @@ def _get_readable_name(self):
 
     @staticmethod
     def get_hash():
-        return "TaipyBase"
+        return NotImplemented
```

Shouldn't this be a raise instead of a return?

---

no I think you can return NotImplemented or raise NotImplementedError